zowe/zowe-cli
# feat(secrets): Core crate to expose Rust modules as library

**What It Does**

This PR implements the following changes for the Secrets SDK:

- Exposes the Rust core logic as a crate named `secrets_core` so that other languages can leverage it to create bindings
  - See [PR 222](https://github.com/zowe/zowe-client-python-sdk/pull/222) on the Python SDK :yum: 
- Adjusts the Rust crate so it can be imported by other Rust-powered Zowe projects (such as the daemon)
- Adds basic unit tests to the `secrets_core` crate

**How to Test**

Build the Secrets SDK package using `npm run build` - compilation should succeed.

**Review Checklist**
I certify that I have:


**Additional Comments**
